pkg/syncer/types/commontypes.go

```diff
@@ -57,32 +57,38 @@ func InitConfigInfo(ctx context.Context) (*ConfigInfo, error) {
 
 // GetVirtualCenterInstance returns the vcenter object singleton.
 // It is thread safe.
-func GetVirtualCenterInstance(ctx context.Context, configTypes *ConfigInfo) (*cnsvsphere.VirtualCenter, error) {
-	onceForVirtualCenter.Do(func() {
-		log := logger.GetLogger(ctx)
+func GetVirtualCenterInstance(ctx context.Context, configTypes *ConfigInfo, reloadConfig bool) (*cnsvsphere.VirtualCenter, error) {
+	log := logger.GetLogger(ctx)
+	vCenterInstanceLock.Lock()
+	defer vCenterInstanceLock.Unlock()
+
+	if !vCenterInitialized || reloadConfig {
```

Comment:
Can you add a log within this scope to say "initializing"?

Reply:
Added.